packages/mds-transactions-api/handlers/get-transactions.ts

```diff
@@ -14,28 +14,154 @@
+           */
+          throw new ValidationError(`Invalid sort direction ${x}`)
+        }
+      }
+    })
+    .query('order_direction')
+
+  const order = column ? { column, direction } : undefined
+
+  return order
+}
+
+/**
+ * Construct URLs given search options & cursor location
+ * @param req Express Request
+ * @param param1 Search options & cursor location
+ */
+const constructUrls = (
+  req: TransactionApiGetTransactionsRequest,
+  { order, ...basicOptions }: TransactionSearchParams
+) => {
+  const url = new URL(`${req.get('x-forwarded-proto') || req.protocol}://${req.get('host')}${req.path}`)
+
+  const basicOptionsUrls = Object.entries(basicOptions).reduce((urlParams, [key, val]) => {
+    if (val) {
+      if (urlParams === '?') return `${urlParams}${key}=${val}`
+      return `${urlParams}&${key}=${val}`
+    }
+
+    return urlParams
+  }, `${url}?`)
+
+  // We can do this because we're assured to have *something* in `basicOptionsUrls` if we're generating a page
+  if (order) {
+    return `${basicOptionsUrls}&order_column=${order.column}&order_direction=${order.direction}`
+  }
+
+  return basicOptionsUrls
+}
 
 export const GetTransactionsHandler = async (
   req: TransactionApiGetTransactionsRequest,
   res: TransactionApiGetTransactionsResponse
 ) => {
   try {
-    const transactions = await TransactionServiceClient.getTransactions({
-      ...parseRequest(req).single({ parser: String }).query('provider_id'),
-      ...parseRequest(req).single({ parser: Number }).query('start_timestamp', 'end_timestamp')
+    const order = getOrderOption(req)
+    const { provider_id, before, after } = parseRequest(req)
+      .single({ parser: String })
+      .query('provider_id', 'before', 'after')
+    const { start_timestamp, end_timestamp, limit = 10 } = parseRequest(req)
+      .single({ parser: Number })
+      .query('start_timestamp', 'end_timestamp', 'limit')
+
+    const { transactions, cursor } = await TransactionServiceClient.getTransactions({
+      provider_id,
+      before,
+      after,
+      start_timestamp,
+      end_timestamp,
+      order,
+      limit
     })
+
     const { version } = res.locals
-    return res.status(200).send({ version, transactions })
+
+    const links: PaginationLinks = {
+      prev: cursor.beforeCursor
+        ? constructUrls(req, {
+            order,
+            provider_id,
+            start_timestamp,
+            end_timestamp,
+            limit,
+            before: cursor.beforeCursor
+          })
+        : null,
+      next: cursor.afterCursor
+        ? constructUrls(req, {
+            order,
+            provider_id,
+            start_timestamp,
+            end_timestamp,
+            limit,
+            after: cursor.afterCursor
+          })
+        : null
+    }
+
+    return res.status(200).send({ version, transactions, links })
   } catch (error) {
     return res.status(500).send({ error: new ServerError(error) })
```

Comment:
Note that the API does not currently handle error throws especially well. I'll write a global error handling middleware in another PR.